### ISSUE TITLE
Feature: Make path to htaccess template configurable

### ISSUE THREAD

**Issue by codemonkey1988** [lochmueller/staticfilecache#40]
The path to the htaccess fluid template should be configurable.

Use-case:
Currently, all header values get escaped by the default fluid behaviour.
This breaks headers like CSP or http/2 server push. 
When using this headers, the template needs to be customized.

**Comment by lochmueller** [comment]
Do you mean this line:
https://github.com/lochmueller/staticfilecache/blob/master/Resources/Private/Templates/Htaccess.html#L25
Perhaps we should add some <f:format.raw>. Sound like a Bug in the current template, right?

**Comment by jdoubleu** [comment]
I'd rather disable escaping for the whole file?
https://usetypo3.com/24-fluid-tips.html#c449
This could break the `.htaccess` sometimes (e.g. when any value contains quotes), even without the user noticing it in first place.

However I'd also enable overriding this template, so others can extend/adjust it to their needs.

**Comment by lochmueller** [comment]
Good point. I will check both (disable escape and templatePath configuration) in the next days/weeks.

**Comment by codemonkey1988** [comment]
That's the line I was talking about. Using the f:format.raw viewhelper fixed that problem for our use-case. But this should be handled in a another issue. I will create another one and can give you a PR after some testing.

Never than less, I think it would be nice, to have the option, to customize the template by using my own one. There are for sure other use-cases where a customized template come in handy.
